### ISSUE TITLE
TypeError: ufunc 'isfinite' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''

### ISSUE THREAD

**Issue by anushiya-thevapalan** [DeepSpectrum/DeepSpectrum#26]
I am trying to extract the features from an audio file. But I am getting the following error. I am unable to figure the exact error. Could you assist in figuring out the issue

```Traceback (most recent call last):
  File "/home/anushiya/anaconda3/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/deepspectrum/backend/plotting.py", line 108, in plot_chunk
    fig.add_axes(spectrogram_axes, id='spectrogram')
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/matplotlib/figure.py", line 1236, in add_axes
    if not np.isfinite(rect).all():
TypeError: ufunc 'isfinite' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/anushiya/pythonenv/ds_virtualenv_2/bin/deepspectrum", line 8, in <module>
    sys.exit(cli())
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/deepspectrum/cli/features.py", line 41, in features
    writer.write_features(configuration.files, extractor, hide_progress=False)
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/deepspectrum/tools/feature_writer.py", line 41, in write_features
    for batch in features:
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/deepspectrum/backend/extractor.py", line 54, in __next__
    return self.extract_features(next(self.images))
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/deepspectrum/backend/extractor.py", line 305, in _batch_images
    for plot_tuple in images:
  File "/home/anushiya/pythonenv/ds_virtualenv_2/lib/python3.7/site-packages/deepspectrum/backend/plotting.py", line 322, in __next__
    return next(self.plots)
  File "/home/anushiya/anaconda3/lib/python3.7/multiprocessing/pool.py", line 748, in next
    raise value
TypeError: ufunc 'isfinite' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''
```

**Comment by mauricege** [comment]
Hi,

thanks for filing this issue.

How did you install DeepSpectrum? Could you also post the versions of your installed python packages? There are currently some dependency related issues I'm working on.

For the time being, you could try to install the pip release of DeepSpectrum which should work. (pip install deepspectrum)

Thanks,
Maurice

**Comment by anushiya-thevapalan** [comment]
Hi, 

Thank you for your response.

I installed using pip. First I cloned the project from the repo, and then as suggested I created a virtualenv.
```
cd DeepSprectrum
pip install .
```

**Comment by anushiya-thevapalan** [comment]
As you suggested, I now tried the following command and it works. Thank you for the quick response
```
pip install deepspectrum
```

**Comment by mauricege** [comment]
This is solved in #25
